src/InnoSetup/Environment.iss

```diff
@@ -384,6 +386,10 @@ begin
         GitUseMirror := False;
         IsGitRecursive := True;
         GitRepository := 'https://jihulab.com/esp-mirror/espressif/esp-idf.git';
+        { Some submodules's submodule using absolute path, redirect url needed}
+        if Exec(GitExecutablePath, + ' config --global url.https://jihulab.com/esp-mirror.insteadOf https://github.com', '', SW_HIDE, ewWaitUntilTerminated, ResultCode) then begin
```

Comment:
The side effect is that, if a customer happens to use git to clone another github repository during the repository cloning, it will also be replaced with a jihulab link and may fail due to a non-existent url.

Of course, since the user chose jihulab, we assume that his access to github is problematic, and the probability of this happening is relatively small 😄

Reply:
`Set local repository config for URL rewriting` Does this configuration will apply to submodules as well?